changelog/6276.improvement.rst

```diff
@@ -0,0 +1 @@
+Allow Rasa to boot when model loading exception occurs. Forward HTTP Error responses to standard log output.
```

Comment:
can you please change the type of the file to `.md` due to our new docsystem? Thanks 🙏

Reply:
My bad, missed the consequence on extension!